procrastinate/schema.py

```diff
@@ -16,6 +21,17 @@ def __init__(self, connector: connector_module.BaseConnector):
     def get_schema() -> str:
         return read_text("procrastinate.sql", "schema.sql")
 
+    @property
+    def version(self):
```

Comment:
Seems to me like a staticmethod `get_version()` would make your life easier in a few places.
You can even implement this property as calling the staticmethod if you want, but you don't even have to.

Reply:
Yep, I was actually considering doing it.